src/pics.c

```diff
@@ -186,6 +186,8 @@ int main_pics(int argc, char* argv[argc])
 		OPT_SELECT('a', enum algo_t, &algo, ALGO_PRIDU, "select Primal Dual"),
 		OPT_SET('M', &sms, "Simultaneous Multi-Slice reconstruction"),
 		OPTL_SET('U', "lowmem", &nuconf.lowmem, "Use low-mem mode of the nuFFT"),
+		OPT_INFILE('X', &nuconf.save_psf_path, "", "Path to save non-cartesian PSF"),
```

Comment:
This would need to be OPT_OUTFILE, as it specifies an output file.

Reply:
Thanks, I pushed the changed the rebased everything to a single commit.